src/Robo/Commands/Blt/DrushCliCommand.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+namespace Acquia\Blt\Robo\Commands\Blt;
+
+use Acquia\Blt\Robo\BltTasks;
+use Acquia\Blt\Robo\Exceptions\BltException;
+use Robo\Contract\VerbosityThresholdInterface;
+
+/**
+ * Defines commands for installing the Drush shell alias and respective executables.
+ */
+class DrushCliCommand extends BltTasks {
+
+  /**
+   * Installs the Drush CLI aliases and depedencies for Drush8/9 command line usage.
+   *
+   * @command blt:init:drush:cli-tools
+   *
+   * @aliases drushtools install-drush-cli-tools
+   */
+  public function installDrushCliTools() {
+    if (!$this->getInspector()->isDrushCliAliasInstalled()) {
+      $config_file = $this->getInspector()->getCliConfigFile();
+      if (is_null($config_file)) {
+        $this->logger->warning("Could not find your CLI configuration file.");
+        $this->logger->warning("Looked in ~/.zsh, ~/.bash_profile, ~/.bashrc, ~/.profile, and ~/.functions.");
+        if (!$created) {
+          $this->logger->warning("Please create one of the aforementioned files, or create the Drush CLI aliases manually.");
+        }
+      }
+      else {
+          $source = $this->getConfigValue('repo.root');
+          $this->redispatchToVendorBin();
+          $this->setupComposerBinPlugin();
+          $this->createNewDrushCliAlias($source);
+      }
+    }
+
+    else {
+      $this->say("<info>The Drush CLI alias and dependencies are already installed.</info>");
+    }
+  }
+
+
+  /**
+   * Install and configure composer bin plugin. 
+   *
+   * @command setup:composer:bin-plugin
+   *
+   * @aliases composer-bin-plugin
+   */
+  public function setupComposerBinPlugin() {
+
+      $this->say('Adding composer vendor bin packages and config...');
+      $this->say('Adding drush 9 binaries and dependencies');
+       $result = $this->taskExec("composer bin drush-9 require drush/drush")
+          ->printOutput(TRUE)
+          ->dir($this->getConfigValue('repo.root'))
+          ->run();
+
+      $this->say('Adding drush 8 binaries');
+       $result = $this->taskExec("composer bin drush-8 require drush/drush:8.1.16")
+          ->printOutput(TRUE)
+          ->dir($this->getConfigValue('repo.root'))
+          ->run();
```

Comment:
if (!$result->wasSuccessful()) {
      throw new BltException("Unable to install Drush 8");
    }